src/features/profile/contactInfo/ContactInfo.tsx

```diff
@@ -1,34 +1,36 @@
+import { Button } from 'hds-react';
 import { useTranslation } from 'react-i18next';
 
+import { ContactInfo as ContactInfoType } from '../types';
 import LabelValuePair from '../../../common/labelValuePair/LabelValuePair';
 import './contactInfo.scss';
 
-export interface ContactInfoProps {
-  name: string;
-  address: string;
-  postalCode: string;
-  municipality: string;
-  phoneNumber: string;
-  emailAddress: string;
-  customerGroup: string;
-  language: string;
-}
+export type ContactInfoProps = {
+  contactInfo?: ContactInfoType;
+  handleEdit(): void;
+};
 
-const ContactInfo = ({
-  name,
-  address,
-  postalCode,
-  municipality,
-  phoneNumber,
-  emailAddress,
-  customerGroup,
-  language,
-}: ContactInfoProps) => {
+const ContactInfo = ({ contactInfo, handleEdit }: ContactInfoProps) => {
   const { t } = useTranslation();
 
+  if (!contactInfo) {
+    return <div>Sinulla ei ole vielä profiilia. Luo uusi profiili tästä.</div>;
```

Comment:
Do we have translations for this statement?

Reply:
Not yet 🤔

Reply:
I moved this to Container level and changed it to English. There needs to be a separate profile creation form. I can start it today, but someone probably has to pick up the work later